webcompat/static/js/lib/bugform.js

```diff
@@ -77,9 +77,14 @@ function BugForm() {
     );
     this.problemType.on("change", _.bind(this.checkProblemTypeValidity, this));
     this.uploadField.on("change", _.bind(this.checkImageTypeValidity, this));
-    this.osField
-      .add(this.browserField)
-      .on("blur input", _.bind(this.checkOptionalNonEmpty, this));
+    this.osField.on(
+      "blur",
+      _.bind(this.checkOptionalNonEmpty, this, this.osField)
+    );
+    this.browserField.on(
+      "blur",
+      _.bind(this.checkOptionalNonEmpty, this, this.browserField)
```

Comment:
If we're going to do it this way, we don't even need to pass in `this.browserField` -- we can just get that from `event.target` down in `checkOptionalNonEmpty`, right?

Reply:
With `this.browserField` we have already set a reference and I feel like this is more readable for someone not knowing the code, but maybe I am missing something?

Reply:
Sure, I guess it's just a question of style preference.